clouddriver/src/integration/java/io/armory/plugin/eap/ClouddriverIntegrationTest.kt

```diff
@@ -0,0 +1,30 @@
+package io.armory.plugin.eap
```

Comment:
I am just wondering if it would make sense to move the integration tests to the **integrationtest**  module?

Reply:
we can also leverage the configuration done in `integration-tests.gradle` which is somehow similar to the one done in `clouddriver.gradle`

Reply:
@cs-simental - I agree with @ovidiupopa07 , we should move the integration tests to the `integration-tests` module.

Reply:
I'll keep the test in the cloud driver integration module as running the other integration test is behaving strangely with git hub actions